tests/plugins/test_mrtmk.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from streamlink.plugins.mrtmk import MRTmk
+
+
+class TestPluginBigo(unittest.TestCase):
```

Comment:
could you do it like in this file instead

https://github.com/streamlink/streamlink/blob/f65274222970cad996bf9d89bdd4f89e8f5d1549/tests/plugins/test_rtpplay.py#L1-L25